pkg/data/management/machinedriver_data.go

```diff
@@ -5,14 +5,16 @@ import (
 	"os"
 	"os/exec"
 	"reflect"
+	goruntime "runtime"
```

Comment:
```suggestion
	"runtime"
```
Nit: runtime is self-explanatory IMO